.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,53 @@
+name: Feature Request
+description: File a feature request
+title: "[Feat]: "
+labels: ["💻 aspect: code", "⭐ goal: addition", "🚦 status: awaiting triage"]
+assignees:
+  - krishdevdb
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
+
+  - type: textarea
+    id: detailed-description
+    attributes:
+      label: Detailed Description
+      description: Add a detailed description of the feature request.
+      placeholder: A detailed description of the feature request.
+      value: "I found this error in this project where ..."
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: contributing
+    attributes:
+      label: Contributing
+      description: You accept that you've read the [Contributing](https://github.com/krishdevdb/.github/blob/main/contributing.md)
```

Comment:
These links are broken now 😅

Reply:
https://github.com/krishdevdb/.github/blob/main/code_of_conduct.md this one is still valid no !👀

Reply:
I mean we shall move them back to this repo